pkg/ddevapp/traefik.go

```diff
@@ -370,6 +370,71 @@ func configureTraefikForApp(app *DdevApp) error {
 		UseLetsEncrypt:  globalconfig.DdevGlobalConfig.UseLetsEncrypt,
 	}
 
+	// The following block reads the project/.ddev/traefik/traefik_dynamic_config_extensions_template.yaml file, fills any template placeholders in it with templateData, and populates the Extension struct. Will later be added to templateData, to be used in generating the final .ddev/traefik/config/project.yaml file. This ultimately allows traefik extension plugins to be seamlessly used in the DDEV Traefik Router on a per-project basis. Add-ons can inject their extensions as-needed
+
+	extensionsYamlFilePath := projectTraefikDir + "/traefik_dynamic_config_extensions_template.yaml"
```

Comment:
Why are we reading a file from the traefik directory instead of reading it from go:embed? There's no reason for this file to be written out is there?